.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -824,6 +824,7 @@ mvn
 Mycheckstyle
 mycompany
 mycompanychecks
+MYCONSTANT
```

Comment:
Really strange that comment was lost in github about this line.

Please use some existing word in name of constant, to avoid extension of whitelist with bad words.

If you like "my...." , try to use "myself".

Reply:
Done.

Reply:
@romani Sir, not really sure what the error means in the circleci.